public/keymaps/m/murcielago_rev1_default.json

```
@@ -0,0 +1,36 @@
+{
+  "keyboard": "murcielago/rev1",
+  "keymap": "default_6a1a08a1",
+  "commit": "6a1a08a162ab1d03e823cb21df3f2e427bd1508c",
```

```suggestion
  "keymap": "default",
  "commit": "fadd3cb4617fe7e48c802c4470a50df36e6c5109",
```

- The `keymap` key no longer has the name collision issue in the API. You can use `default` as the value here.
- I get the commit hashes by running `git log -1 --pretty=format:"%H" -- keyboards/<keyboard>/keymaps/default/keymap.c` from QMK's current `master` branch, e.g.
  ```shell
  git checkout master
  git pull upstream master
  git log -1 --pretty=format:"%H" -- keyboards/murcielago/rev1/keymaps/default/keymap.c
  ```

---

Ok thanks, I though that it was just chosen randomly!